### ISSUE TITLE
Lina - Setup Jabber

### ISSUE THREAD

**Issue by cassininazir** [creativedisturbance/podcasts#70]
Please confirm that you are able to use Jabber. Review the document at [CD 1.3 Working Remotely](https://docs.google.com/document/d/1NP-TLJ38swdwy3bZrvcW4umn3Dmy1q5y8RrTfkkNbuE/edit) for instructions.

**Complete by Monday, May 25**

**Comment by coreysmart92** [comment]
@linamoon Please do this before Monday, and post any issues here. Thank you

**Comment by linamoon** [comment]
@coreysmart92 @cassininazir I emailed Brian at brian.merlo@utdallas.edu on Monday to be added to the server, but I haven't received a response back.

**Comment by cassininazir** [comment]
I think I added your name on the original list. Did you go thru the setup and it not work?

Sent from my iPhone

On May 21, 2015, at 5:15 PM, Lina Moon <notifications@github.com<mailto:notifications@github.com>> wrote:

@coreysmart92https://github.com/coreysmart92 @cassininazirhttps://github.com/cassininazir I emailed Brian at brian.merlo@utdallas.edumailto:brian.merlo@utdallas.edu on Monday to be added to the server, but I haven't received a response back.

�
